singer-connectors/target-snowflake/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-target-snowflake==1.5.0
\ No newline at end of file
+-e git+https://github.com/transferwise/pipelinewise-target-snowflake.git@master#egg=pipelinewise-target-snowflake
```

Comment:
can we change it back to pypi release version?

Reply:
Yes, this is just temporary, cuz the latest changes in target-snowflake plugin aren't released yet to PyPi